packages/tag/index.demo.tsx

```diff
@@ -1,14 +1,30 @@
-import { h, Component } from 'skatejs';
+import { h, Component, prop } from 'skatejs';
 import { Tag } from './Tag';
+import { TagSelector } from './TagSelector';
 
 export class Demo extends Component<void> {
   static get is() { return 'bl-tag-demo' }
+  static get props() {
+    return {
+      tags: prop.array()
+    }
+  }
+
+  tags = ['milk','bread','chocolate'];
 
   private tagClose() {
-    console.log('Tag close emitted', this)
+    console.log( 'Tag close emitted', this )
+    const element = this;
+    element.parentNode.removeChild( element );
+  }
+
+  private tagChange( event ) {
+    console.log( 'Tag change:', event.detail.tags );
+    this.tags = [].concat( event.detail.tags );
```

Comment:
you can do shorthand via spread ( shallow copy )
`this.tags = [ ...event.detail.tags ];`